src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -191,24 +191,14 @@ public void CreateTagger_tracker_should_be_distinct_per_file()
             tagger1.Should().NotBe(tagger2);
         }
 
-        [TestMethod]
-        public void TaggerAdded_SonarErrorDataSourceIsNotified()
-        {
-            CreateTagger(jsContentType);
-            var trackers = provider.ActiveTrackersForTesting.ToArray();
-            trackers.Count().Should().Be(1); // sanity check
-
-            mockSonarErrorDataSource.Verify(x => x.AddFactory(trackers[0].Factory), Times.Once);
-        }
-
```

Comment:
The behaviour being tested is now internal to the tracker. There's a new more comprehensive unit test in the tracker tests.